scheduled-jobs/build/oc-mirror-quay-dev/Jenkinsfile

```diff
@@ -1,68 +1,58 @@
-                    name: 'TARGET_NODE',
-                    description: 'Jenkins agent node',
-                    $class: 'hudson.model.StringParameterDefinition',
-                    defaultValue: 'openshift-build-1'
-                ],
-                [
-                    name: 'MAIL_LIST_SUCCESS',
-                    description: 'Success Mailing List',
-                    $class: 'hudson.model.StringParameterDefinition',
-                    defaultValue: [
-                        'aos-team-art@redhat.com',
-                    ].join(',')
-                ],
-                [
-                    name: 'MAIL_LIST_FAILURE',
-                    description: 'Failure Mailing List',
-                    $class: 'hudson.model.StringParameterDefinition',
-                    defaultValue: [
-                        'aos-team-art@redhat.com',
-                    ].join(',')
-                ],
-
-            ]
-        ],
-        disableConcurrentBuilds()
-    ]
-)
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '360'
+                )
+            ),
+            [
+                $class : 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    [
+                        name: 'MOCK',
+                        description: 'Mock run to pickup new Jenkins parameters?',
+                        $class: 'hudson.model.BooleanParameterDefinition',
+                        defaultValue: false,
+                    ],
+                    commonlib.ocpVersionParam('BUILD_VERSION', '4'),
+                    [
+                        name: 'TARGET_NODE',
+                        description: 'Jenkins agent node',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: 'openshift-build-1'
+                    ],
+                    [
+                        name: 'MAIL_LIST_SUCCESS',
+                        description: 'Success Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'aos-team-art@redhat.com',
+                        ].join(',')
+                    ],
+                    [
+                        name: 'MAIL_LIST_FAILURE',
+                        description: 'Failure Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'aos-team-art@redhat.com',
+                        ].join(',')
+                    ],
 
-node(TARGET_NODE) {
-    checkout scm
-
-    def commonlib = load( "pipeline-scripts/commonlib.groovy")
-    commonlib.initialize()
+                ]
+            ],
+            disableConcurrentBuilds()
+        ]
+    )
```

Comment:
And then
```
commonlib.checkMock()
```

Reply:
as above. not going to keep updating this